src/ps1/ps1_cardman.c

```diff
@@ -164,3 +177,13 @@ int ps1_cardman_get_idx(void) {
 int ps1_cardman_get_channel(void) {
     return card_chan;
 }
+
+void ps1_cardman_set_gameid(const char* game_id) {
+    memcpy(card_game_id, game_id, 0x10);
+    card_idx = IDX_GAMEID;
+    card_chan = CHAN_MIN;
+}
+
+void ps1_cardman_get_gametext(char* const game_text) {
+    memcpy(game_text, card_game_id, 0x10);
```

Comment:
this function should just return `card_game_id` as a pointer, not do a copy
if it has to do a copy, then it must also take size of the passed `game_text` buffer and use a safe string copy function (strlcpy)